typed-builder-macro/src/mutator.rs

```diff
@@ -52,7 +52,7 @@ impl ApplyMeta for MutatorAttribute {
 }
 
 impl Parse for Mutator {
-    fn parse(input: ParseStream) -> syn::Result<Self> {
+    fn parse(input: ParseStream<'_>) -> syn::Result<Self> {
```

Comment:
This seems redundant. Is this because of that `#![forbid(rust_2018_idioms)]` thing?

Reply:
Yes. This lets to see at a glance where a borrowing occurs